hack/ci/run-e2e-tests.sh

```diff
@@ -36,7 +47,5 @@ else
 fi
 pushElapsed kind_kubermatic_setup_duration_milliseconds $beforeKubermaticSetup
 
-echodate "Done setting up Kubermatic in kind"
-
 echodate "Running conformance tests"
-KUBERMATIC_NO_WORKER_NAME=true ./hack/ci/run-conformance-tests.sh
```

Comment:
Is there somewhere else where this was set to false?

Reply:
No, it was only used in the one if statement that I removed in this PR as well.